test/helpers/storageQueries.ts

```diff
@@ -49,7 +49,7 @@ export async function processAllStorage(
       prefixes.map(async (prefix) =>
         limiter.schedule(async () => {
           let startKey = null;
-          while (true) {
+          loop: while (true) {
```

Comment:
PS: I don't like this syntax :p it makes it harder to read as you can now have a non-keyword starting word that is not a variable neither

Reply:
ok, will remove in future versions. I added the syntax to make it clearer, which it sounds like it clearly isnt doing.